### ISSUE TITLE
What is this line for ?

### ISSUE THREAD

**Issue by lchauvin** [lchauvin/Slicer-CircleFit#3]
Hi @tokjun ,
I was wondering what is this line for ?
https://github.com/lchauvin/Slicer-CircleFit/blob/af7fbe48d75fe495651b8f6fc34df9d22b705bbe/CircleFit/CircleFit.cxx#L319

If I understand correctly, you're checking that distance between p1 and p2, and p2 and p3, is below 10mm ? Why ?

-Laurent

**Comment by tokjun** [comment]
Just to make sure that the distances between them are sufficient. They could never be less than the diameter of the spheres.

**Comment by lchauvin** [comment]
Oh, okay.
Should I make it a parameter of the CLI ?

**Comment by tokjun** [comment]
Yes, but it should be synchronized with diameter parameter for the marker detection. Otherwise it will confuse the users.
